modules/pel/peltool/peltool.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
```

Comment:
probably better to use /usr/bin/env python so in virtual environments it will use the correct python version

Reply:
> probably better to use /usr/bin/env python so in virtual environments it will use the correct python version

Done